### PR TITLE
seeding RHDE

### PR DESCRIPTION
**Pull request by kalexand-rh**
http://file.rdu.redhat.com/kalexand/rhde-docs/overview/rhde-overview.html

New files:
- _attributes/common-attributes.adoc
- modules/about-rhde.adoc
- overview/rhde-overview.adoc
- snippets/developer-preview.adoc

Changed files:
- _distro_map.yml
- _topic_map.yml
- .travis.yml

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@ShaunaDiaz, will you PTAL at the rendered topic and the handful of added .adoc files that feed into it? This should be a copy of the [published Device Edge docs](https://access.redhat.com/documentation/en-us/red_hat_device_edge/4) with one extra heading from the module I added.

**Review by ShaunaDiaz — COMMENTED** [review]
The two images can be replaced with Jess' 311 series updates. https://github.com/redhataccess/documentation-svg-assets/tree/master/for-web/RHDE/311_RHDE

File: _topic_map.yml [hunk 1]
```diff
@@ -22,259 +22,10 @@
 # topic groups and topics on the main page.
 
 ---
-Name: About
-Dir: welcome
-Distros: openshift-acs
+Name: Overview
```
Comment (gaurav-nelson) [inline _topic_map.yml:25]: This makes the content appear as Overview > Overview

Maybe we can change it to:

- About > Overview
- Welcome > Overview

File: .travis.yml [hunk 2]
```diff
@@ -44,7 +44,7 @@ jobs:
       env:
         - CAN_FAIL=true
       language: minimal
-      if: branch IN (rhacs-docs, rhacs-docs-3.74, rhacs-docs-3.73, rhacs-docs-3.72, rhacs-docs-3.71)
+      if: branch IN (rhde-docs)
```
Comment (gaurav-nelson) [inline .travis.yml:47]: **thought:** For L42 - L49 and L54

If we don't need Netlify preview builds for this distro we can remove these lines.

**Pull request merged**